integration_tests/dbt_project.yml

```diff
@@ -16,6 +16,7 @@ vars:
   # shopify_using_product_variant_media: true
```

Comment:
Let's also add in the new variables which are disabled by default to be turned on during docs generation.